# remove references to win32 code we don't build
# patch from fedora
--- org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java	2006-07-27 11:35:22.000000000 +0200
+++ org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java	2006-08-22 10:22:49.000000000 +0200
@@ -131,11 +131,6 @@
    					"azbpcorepatcher", 
    					"CorePatcher",
 					"true" },
-	   		{	 PluginManagerDefaults.PID_PLATFORM_CHECKER, 
-   					"org.gudy.azureus2.platform.win32.PlatformManagerUpdateChecker", 
-   					"azplatform2", 
-   					"azplatform2",
-					"true" },
 	   		//{	 PluginManagerDefaults.PID_JPC, 
 				//	"com.aelitis.azureus.plugins.jpc.JPCPlugin", 
 				//	"azjpc", 
